src/action.c

```diff
@@ -633,6 +633,10 @@ show_menu(struct server *server, struct view *view,
 		return;
 	}
 
+	/* need to be current to be useful */
+	create_client_list_combined_menu(menu->server);
+	create_client_send_to_menu(menu->server);
```

Comment:
This can be removed completely as its already handled in `menu_open_root()`.

Reply:
It didn't update properly (latest running apps) without this. There is hardly any time spent in the creating
the menu items, basically spinning through a couple of in memory structures.

Reply:
Interesting, I was pretty sure I tested with out these and it didn't work, but I just tried it and
we can remove those and the need for public.